<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RoryMcCann" ignored="4" name="Default test(failed)" tests="1" failures="1" timestamp="03 Dec 2022 04:06:28 GMT" time="0.147" errors="0">
  <testcase name="tc5" time="0.016" classname="ReRunFailedTestCases.ManualRerun1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at ReRunFailedTestCases.ManualRerun1.tc5(ManualRerun1.java:30)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc5 -->
  <testcase name="tc4" time="0.0" classname="ReRunFailedTestCases.ManualRerun1">
    <ignored/>
  </testcase> <!-- tc4 -->
  <testcase name="tc3" time="0.0" classname="ReRunFailedTestCases.ManualRerun1">
    <ignored/>
  </testcase> <!-- tc3 -->
  <testcase name="tc2" time="0.0" classname="ReRunFailedTestCases.ManualRerun1">
    <ignored/>
  </testcase> <!-- tc2 -->
  <testcase name="tcl" time="0.0" classname="ReRunFailedTestCases.ManualRerun1">
    <ignored/>
  </testcase> <!-- tcl -->
</testsuite> <!-- Default test(failed) -->
